runtime/benches/append_vec.rs

```diff
@@ -1,35 +1,27 @@
 #![feature(test)]
-
-extern crate rand;
 extern crate test;
 
-use bincode::{deserialize, serialize_into, serialized_size};
 use rand::{thread_rng, Rng};
-use solana_runtime::append_vec::{
-    deserialize_account, get_serialized_size, serialize_account, AppendVec,
-};
+use solana_runtime::append_vec::AppendVec;
 use solana_sdk::account::Account;
 use solana_sdk::pubkey::Pubkey;
-use std::env;
 use std::fs::{create_dir_all, remove_dir_all};
-use std::io::Cursor;
 use std::path::PathBuf;
-use std::sync::atomic::{AtomicUsize, Ordering};
-use std::sync::{Arc, RwLock};
+use std::sync::{Arc, Mutex};
+use std::thread::sleep;
 use std::thread::spawn;
+use std::time::Duration;
 use test::Bencher;
 
-const START_SIZE: u64 = 4 * 1024 * 1024;
-const INC_SIZE: u64 = 1 * 1024 * 1024;
-
-macro_rules! align_up {
-    ($addr: expr, $align: expr) => {
-        ($addr + ($align - 1)) & !($align - 1)
-    };
+fn test_account(ix: usize) -> Account {
```

Comment:
Please only use `ix` as a shorthand for `instruction` in the codebase. This one is especially confusing because it uses `ix` as a shorthand for an index to an instruction. For indexes, use either `index` or `i`. Can you update the many uses `ix` in this PR?